src/app/core/services/data-address-mapper.ts

```diff
@@ -13,7 +12,7 @@ export class DataAddressMapper {
       | AssetDatasourceFormValue
       | ContractAgreementTransferDialogFormValue
       | undefined,
-  ): DataAddressDto {
+  ): Record<string, string> {
     switch (formValue?.dataAddressType) {
       case 'Custom-Data-Address-Json':
         return JSON.parse(formValue.dataDestination?.trim()!!);
```

Comment:
check if the placeholders for the custom data address still contain the nested "properties" field.

The new data address json does not contain the nested properties field anymore: https://user-images.githubusercontent.com/105237007/249055199-c05facb3-2b49-494b-b7c0-06c176d231c0.png